test/functional/specs/Identity/getIdentityAfterCollectCall.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
Can you create a test rail number for this test?

Reply:
Didn’t we say we won’t use that anymore?

Reply:
I don't remember any formal decision, so I've created a testrail case for each of the functional tests that were missing one.